HelpSource/Classes/TempoClock.schelp

```diff
@@ -247,6 +247,38 @@ t.beats2secs(0) // how many seconds after startup did beat 0 occur?
 method::secs2beats
 Converts absolute strong::seconds:: to absolute beats. Only works for times in the current tempo. If the tempo changes any computed time in future will be wrong.
 
+method::sync
+Gradually synchronize one or more clocks to the same downbeat and tempo.
```

Comment:
This method only synchronizes one clock (the current one), so this is a misleading statement.